test/server/cards/02_SHD/units/MandalorianWarrior.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Mandalorian Warrior', function () {
+    integration(function () {
+        describe('Mandalorian Warrior\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['mandalorian-warrior'],
+                        groundArena: ['protector-of-the-throne', 'battlefield-marine'],
+                    },
+                    player2: {
+                        groundArena: ['snowspeeder']
+                    }
+                });
+            });
+
+            it('should give an experience to mandalorian unit', function () {
+                // add experience to a mandalorian
+                this.player1.clickCard(this.mandalorianWarrior);
+                this.player1.clickPrompt('Give 1 experience to another Mandalorian unit');
```

Comment:
Could you add `expect(this.player1).toHavePassAbilityButton();`. I forgot about that one